### PR TITLE
E2E volunteer signup

### PR DESCRIPTION
**Pull request by treykris**
Links
-----
- Server repo PR: https://github.com/UPchieve/server/pull/399

Description
-----------
- Added e2e tests for photo uploading and adding references

Developer self-review checklist
-------------------------------
- [x] Potentially confusing code has been explained with comments
- [x] No warnings or errors have been introduced; all known error cases have been handled
- [x] Any appropriate documentation (within the code, README.md, docs, etc) has been updated
- [ ] All edge cases have been addressed

### CODE REVIEW

**Comment by jkeat** [comment]
@treykris should the base branch of this be `volunteer-signup`?

**Comment by treykris** [comment]
Updated!

File: tests/e2e/specs/volunteer-dashboard-tests.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  describe("Open Volunteer", () => {
+    describe("Photo ID upload", () => {
+      beforeEach(function() {
+        cy.login(this.openVolunteer);
+        cy.server();
+        cy.visit("/");
+
+        cy.contains("Proof of identity").click();
+
+        cy.get("input[type=file]").attachFile("images/placeholder.jpg");
```
Comment (jkeat) [inline tests/e2e/specs/volunteer-dashboard-tests.js:15]: This is nice 👍 👍

File: tests/e2e/specs/volunteer-dashboard-tests.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+      });
+
+      it("Should upload a photo id", function() {
+        cy.route("/api/user/volunteer-approval/photo-url", {
+          success: true,
+          message: "AWS SDK S3 pre-signed URL generated successfully",
+          uploadUrl: "noop"
```
Comment (jkeat) [inline tests/e2e/specs/volunteer-dashboard-tests.js:22]: Is this just a random string that fails on client PUT?

Comment (treykris) [reply tests/e2e/specs/volunteer-dashboard-tests.js:22]: Yup, it just resolves to localhost:8080/noop
